src/mito2/src/sst/parquet/helper.rs

```diff
@@ -109,5 +114,14 @@ pub async fn fetch_byte_ranges(
         .await?
         .into_iter()
         .map(|buf| buf.to_bytes())
-        .collect::<Vec<_>>())
+        .collect::<Vec<_>>();
+
+    trace!(
+        "Fetch {} bytes from '{}', cost: {:?}",
+        total_size,
+        file_path,
+        start.elapsed()
+    );
```

Comment:
We already have a timer in the caller of this method. Can we move this log to there?

Reply:
See
https://github.com/GreptimeTeam/greptimedb/blob/9175fa643d5345fa62a1dbbb8d7935c7de47c9da/src/mito2/src/sst/parquet/row_group.rs#L292-L304

Reply:
```rust
                trace!(
                    "Write cache miss for key '{}' (region_id='{}', file_id='{}', row_group_idx='{}') and fetch {} bytes from file '{}' in read cache or object store, cost: {:?}",
                    key,
                    self.region_id,
                    self.file_id,
                    self.row_group_idx,
                    total_size,
                    self.file_path,
                    timer.stop_and_record()
                );
```